ndn_python_repo/handle/sync_command_handle.py

```diff
@@ -2,14 +2,12 @@
 import logging
 from ndn.app import NDNApp
 from ndn.encoding import Name, NonStrictName, DecodeError, Component, parse_data
-from ndn.types import InterestNack, InterestTimeout
 from . import ReadHandle, CommandHandle
 from ..command import RepoCommandRes, RepoCommandParam, SyncParam, SyncStatus, RepoStatCode
 from ..utils import concurrent_fetcher, PubSub, PassiveSvs, IdNamingConv
 from ..storage import Storage
-from typing import Optional, Tuple, List, Dict
+from typing import Dict
```

Comment:
nit: `Dict` can also be removed. Python 3.9+ allows to use `dict` as the type name without importing anything.